README.md

```diff
@@ -1,2 +1,82 @@
-# appcasas-ui
-UI - App Casas
+<img src="./src/assets/logo.png" alt="logo home">
+
+<h3 align="center">App Home</h3>
+
+  <p align="center">
+    This repository contains the UI for a real estate application. 
+    <br />
+    <br />
+
+<details>
+  <summary>Summary</summary>
+  <ol>
+    <li>
+      <a href="#about-the-ui">About the UI</a>
+    </li>
+    <li>
+      <a href="#technologies">Technologies
+      </a>
+    </li>
+     <li>
+      <a href="#pre-requirements-and-instalation">Pre-Requirements and Instalation
+      </a>
+    </li>
+     <li>
+      <a href="#contact">Contact
+      </a>
+    </li>
+    
+
+  </ol>
+</details>
+
+## About the UI
+The UI is the visual part of the project, which allows the user to interact with the website's functionalities, which in turn communicates with the backend and database.
```

Comment:
I believe the last part of the sentence could be re-written like this:

"which will eventually comunicate with the backend and database."

Reply:
Thank you for the suggestion, I will fix it.